### ISSUE TITLE
Lint all usages of CSS variables with/without the Tab's being active

### ISSUE THREAD

**Issue by phoenisx** [willofindie/vscode-cssvar#76]
@phoenisx this is amazing. thank you so much!! 

one question, i noticed that i need to open (and focus) the erroring file in order to find the errors. i had suggested the problems tab idea because i was hoping i would be able to see errors in closed workspace files so that i know which files i need to open and fix. is that impossible?

https://user-images.githubusercontent.com/175330/191370815-4049e181-32b0-4c7c-b931-d421d29e3fc8.mp4

_Originally posted by @jjenzz in https://github.com/willofindie/vscode-cssvar/issues/73#issuecomment-1252951874_

**Comment by jjenzz** [comment]
@phoenisx in response to https://github.com/willofindie/vscode-cssvar/issues/73#issuecomment-1253164323, could a "validate on save" option work? for example, if i am in a file type that the extension watches and i remove a var, hitting `CMD+S` would activate linter for all files and list issues in problems tab?

**Comment by jjenzz** [comment]
yes, something like the former but not sure if continually watching is necessary if it is going to cause the cpu issues you had described before. 

i was thinking that when i remove a var from any of the files listed in `cssvar.files` and then save, the save could trigger a lint check across all files (no watching) and list errors in problems tab. then, when you open the file to fix the error the lint check can behave as it does now - on focused file only because it is the only file that changed.

**Comment by jjenzz** [comment]
on second thoughts, i wonder if any of this is really needed given my comment here https://github.com/willofindie/vscode-cssvar/issues/77#issuecomment-1254859450

**Comment by phoenisx** [comment]
I still feel adding a watch mode to this extension would be great. Since this extension now already supports linting.
Using a separate library just to enable CSS custom property lints would be too much if they are not using `stylelint` or any other lint system. 

Anyways, that's an option left for the users to decide!! 😉
